dev/operator_local.sh

```diff
@@ -19,11 +19,23 @@ set -euo pipefail
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. >/dev/null && pwd)"
 
+kill $(pgrep -f rerun) >/dev/null 2>&1 || true
+
+eval $(python3 $ROOT/manager/cluster_config_env.py "$ROOT/dev/config/cluster.yaml")
+
+python3 $ROOT/manager/update_cli_config.py "$HOME/.cortex/cli.yaml" "default" "http://localhost:8888" "$CORTEX_AWS_ACCESS_KEY_ID" "$CORTEX_AWS_SECRET_ACCESS_KEY"
+
+cp -r $ROOT/dev/config/cluster.yaml ~/.cortex/cluster-local.yaml
+
+if [ "$CORTEX_DISABLE_TELEMETRY" = "true" ]; then
```

Comment:
you asked us to remove this from our bash_profile, is this still being used?

Reply:
I actually had deleted it locally and not yet committed when I asked you to remove it. But since then I decided it might be ok to keep it as well, just to provide us another way to disable it in dev. However it still should not be the default for us (and therefore not in our bash profiles), since we now have the dev telemetry environment.